cmake/popular-common.cmake

```diff
@@ -36,6 +41,9 @@ endif()
 
 vk_add_library(light_common OBJECT ${LIGHT_COMMON_SOURCES})
```

Comment:
Maybe add here something like `if (COMPILE_RUNTIME_LIGHT)`? Because it starts require libc++ to comiple project with old runtime